File: FirstBitSolutions/C Module/Assignments/Assignment 6/Func_Assignment_3/FunctionsType1/Ques4.c
//4 Check the given number is prime or not.
//Input: n = 7
//Output: Prime
#include<stdio.h>
void Prime();//Declaration
void main()
{
	Prime();//Call		
}//main ends here
void Prime(){
		int no,isPrime =1,i;
	printf("Enter a number:");
	scanf("%d",&no);
	for(i=2; i<=no/2; i++) {
		if(no%i==0) {
			isPrime=0;
			break;
		}	
	}
	if(isPrime==1) {
		printf("This is  prime");
	} else {
		printf("This is not prime");
	}
	
}//Prime ends here

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 3/Ques4.java
import java.util.Scanner;
class Ques{ 
public static void main(String args[]){ 
	int no,isPrime =1,i;
	Scanner sc=new Scanner(System.in);
	System.out.printf("Enter a number:");
	no=sc.nextInt();
	for(i=2; i<=no/2; i++) {
		if(no%i==0) {
			isPrime=0;
			break;
		}	
	}
	if(isPrime==1) {
		System.out.printf("This is  prime");
	} else {
		System.out.printf("This is not prime");
	}

 }
}